lsp/ARCHITECTURE.md

```diff
@@ -1,4 +1,18 @@
-# Architechture
+# Architecture
+
+The application consists of two components: the language server, and the language services.
+
+The **Language Server** is the containing application. It hosts language services, implements the Language server protocol (LSP), and queries information from language services in order to resepond to LSP requests.
```

Comment:
The server responds to queries. The client requests from the server. 

This definition makes it sound like the server is the client.